src/node/build/bundle.ts

```diff
@@ -139,6 +139,12 @@ export async function bundle(
                   ) {
                     return 'framework'
                   }
+                  if (/(^|\/)config\.([cm]?js|ts|json)$/.test(id)) {
+                    return 'framework'
+                  }
+                  if (/(^|\/)vitepress\/package.json$/.test(id)) {
```

Comment:
this seems like hardcoding it just to support our config?

Reply:
This has been reverted.